### DISCUSSION TITLE
Create discussion for release

### DISCUSSION THREAD

**Discussion by krupitskas** [ncipollo/release-action#36] [Ideas]
I've just found such an option for release, is it possible to add it as bool input?

![image](https://user-images.githubusercontent.com/5683877/113897346-731b6180-97d3-11eb-9580-1f77a4cb9844.png)

**Comment by ncipollo** [comment]
Neat! I didn't know that was a thing, must be somewhat new.

Yeah, I believe we can set this via the API: https://docs.github.com/en/rest/reference/repos#update-a-release

The property name is `discussion_category_name`.

**Comment by ncipollo** [comment]
I just pushed a new release which includes a new argument: `discussionCategory`. You can set this to the category you would like to create the discussion in. It must match an existing category (the build will fail if it does not).
